packages/node_modules/@ciscospark/media-engine-webrtc/test/unit/spec/engine.js

```diff
@@ -179,7 +191,7 @@ browserOnly(describe)('media-engine-webrtc', function () {
                   engine.stop();
                 });
 
-                step(`initiates a session with audio=${audioStart} and video=${videoStart}`, () => {
+                (((browser.chrome && (flakyInChrome || brokenInChrome)) || (browser.firefox && brokenInFirefox)) ? it.skip : step)(`initiates a session with audio=${audioStart} and video=${videoStart}`, () => {
```

Comment:
This may be more readable broken out into a few lines or variables, .e.g., `skipChrome = browser.chrome && (flaklyInChrome || brokenInChrome);`

Reply:
Yea, I just copied what was existing, but I'll see if I can make it a bit more readable

Reply:
Fixed